locales/en.json

```diff
@@ -4590,7 +4590,7 @@
     "769": "Sandygast",
     "770": "Palossand",
     "771": "Pyukumuku",
-    "772": "Type: Null",
+    "772": "TypeNull",
```

Comment:
This, along with the Tapu mons, doesn't look correct, is this how it is in-game?

Reply:
The correct name is "Type: Null" (https://www.pokemon.com/us/pokedex/type-null), but I changed it to "TypeNull" to be on the safe side so that there are no errors.

Reply:
Oh. The Tapu Mons is a typical case of "finding and replacing spaces". I'll put the space right back in

Reply:
Special characters and spaces aren't a problem for PA, the main problem there lies when the encoding is outside of UTF-8, because of how the json files themselves are encoded
This is supplemented elsewhere by giving python unicode strings (the "\u..." you see here and there) to represent the properly encoded string representation

Back on topic, references to this are usually done in a string, such as in filters: `monsters: ["Type: Null"]`

Reply:
Tapu mons changed with spaces. The Type: Null mon ... you can decide, should I use the original name (Type: Null) or leave it that way (TypeNull)?

Reply:
change it to `Type: Null` please 😄

Reply:
done :)